core/templates/pages/classroom-page/classroom-page.module.ts

```diff
@@ -52,7 +51,6 @@ import { ClassroomDomainConstants } from
     OppiaAngularRootComponent
   ],
   providers: [
-    AppConstants,
```

Comment:
Also I am not sure why this was needed here. Any ideas? /cc @bansalnitish @srijanreddy98

Reply:
I am not really sure! @bansalnitish any idea?